<!-- -->
<launch>
	<arg name="points_node" default="/points_raw" /><!--CHANGE THIS TO READ WHETHER FROM VSCAN OR POINTS_RAW -->
	<arg name="publish_ground" default="true" />
	<arg name="publish_filtered" default="true" />

	<!-- rosrun lidar_tracker vscan_filling -->
	<node pkg="lidar_tracker" type="vscan_filling" name="vscan_filling" />

	<!-- rosrun lidar_tracker euclidean_cluster _points_node:="" -->
	<node pkg="lidar_tracker" type="euclidean_cluster" name="euclidean_cluster">
		<param name="points_node" value="$(arg points_node)" />
		<param name="publish_ground" value="$(arg publish_ground)" /><!--GROUND FILTERING LIKELY WONT WORK ON VSCAN (for obvious reasons)-->
		<param name="publish_filtered" value="$(arg publish_filtered)" /><!--POINTS FILTERING LIKELY WONT WORK ON VSCAN (for obvious reasons)-->
	</node>

</launch>

